Cargo.toml

```diff
@@ -28,10 +28,13 @@ schannel = "0.1.17"
 
 [target.'cfg(not(any(target_os = "windows", target_os = "macos", target_os = "ios")))'.dependencies]
 log = "0.4.5"
-openssl = "0.10.29"
-openssl-sys = "0.9.55"
+# openssl = "0.10.29"
+openssl = { git = "https://github.com/CyberHive/rust-openssl.git"}
+# openssl-sys = "0.9.55"
+openssl-sys = { git = "https://github.com/CyberHive/rust-openssl.git"}
```

Comment:
Both are pointing at the same `rust-openssl` repo; is that right?

Reply:
Yes, the repo is a workspace containing several crates
```
[workspace]
members = [
    "openssl",
    "openssl-errors",
    "openssl-macros",
    "openssl-sys",
    "systest",
]
```